From e46aa21cf20f6ef05ebe3ef3d95b662a7973f8d6 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 30 Mar 2016 16:43:19 +0200
Subject: [PATCH] man: tc-u32: Minor syntax fix

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1275426
Upstream Status: iproute2.git commit b487954d5baf9

commit b487954d5baf95b4402456964eaad76dcb54db82
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Mar 4 12:52:39 2016 +0100

    man: tc-u32: Minor syntax fix

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/tc-u32.8 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/man/man8/tc-u32.8 b/man/man8/tc-u32.8
index 47c8f2d..691f53c 100644
--- a/man/man8/tc-u32.8
+++ b/man/man8/tc-u32.8
@@ -370,6 +370,7 @@ then allows to match various header fields:
 .RS
 .TP
 .BI src " ADDR"
+.TQ
 .BI dst " ADDR"
 Compare Source or Destination Address fields against the value of
 .IR ADDR .
-- 
1.8.3.1

